_data/ballerinashellswanlake.yml

```
@@ -7,16 +7,19 @@
     - title: Starting the Shell
       url: /learn/ballerina-shell/#starting-the-shell
       active: starting-the-shell
-    - title: Stopping the Shell
-      url: /learn/ballerina-shell/#stopping-the-shell
-      active: stopping-the-shell
+    - title: Available commands
```

```suggestion
    - title: Available Commands
```
Please use title case for all titles.

---

Updated in 2a10777065a5455f4155b3fbdd75b7a70f169c75